Source/Core/DolphinQt/Config/Graphics/GeneralWidget.cpp

```diff
@@ -131,6 +137,8 @@ void GeneralWidget::CreateWidgets()
   m_options_layout->addWidget(m_show_messages, 0, 1);
   m_options_layout->addWidget(m_show_ping, 1, 1);
 
+  m_options_layout->addWidget(m_reset_settings, 2, 0, 1, 2);
```

Comment:
I think this is a super weird place for the button. Maybe it should be in the far bottom-left of the window?